src/components/AppComponent.jsx

```diff
@@ -97,6 +98,7 @@ class AppComponent extends Component {
       width,
     } = this.props;
     const isLoggedIn = !!auth.currentUser;
+    const userName = auth.currentUser ? auth.currentUser.displayName : '';
```

Comment:
Theoretisch könntest du hier anstelle von `auth.currentUser ?` auch `isLoggedIn ?` verwenden. Ich weiss nicht, ob das stabiler wäre...

Reply:
isLoggedIn, ist ein boolean ob der user eingeloggt ist, wir möchten ja den namen verwenden um diesen anzuzeigen

Reply:
Die Idee dahinter war eher folgende Anpassung:
```suggestion
    const userName = isLoggedIn ? auth.currentUser.displayName : '';
```
Mit _isLoggedIn_ hast du ja sicher schon einen Boolean für den Entscheid. Aber das ist nur verschönerung, wenn überhaupt.